src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts

```diff
@@ -0,0 +1,124 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Sku } from '@azure/arm-postgresql/src/models';
+import { AzureWizardPromptStep, IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { ext } from '../../../../extensionVariables';
+import { localize } from '../../../../utils/localize';
+import { nonNullProp } from '../../../../utils/nonNull';
+import { IPostgresServerWizardContext } from '../IPostgresServerWizardContext';
+
+class SkuOption {
+    label: string
+    sku: Sku
+}
+
+export class PostgresServerSkuStep extends AzureWizardPromptStep<IPostgresServerWizardContext> {
+    public postgresDefaultStorageSizeMB: number = 51200;
+
+
+    public async prompt(wizardContext: IPostgresServerWizardContext): Promise<void> {
+
+        const placeHolder: string = localize('selectPostgresSku', 'Select the Postgres SKU and options.');
+
+        wizardContext.sku = (await ext.ui.showQuickPick(this.getPicks(), { placeHolder })).data;
+
+    }
+
+    public shouldPrompt(wizardContext: IPostgresServerWizardContext): boolean {
+        return wizardContext.sku === undefined;
+    }
+
+    public async getPicks(): Promise<IAzureQuickPickItem<Sku>[]> {
+        const options: IAzureQuickPickItem<Sku>[] = [];
+        availableSkus.forEach(option => {
+            options.push({ label: localize(nonNullProp(option.sku, 'name'), option.label), data: option.sku });
+        });
+        return options;
+    }
+}
+
+const availableSkus: Array<SkuOption> = [
+    {
+        label: "Basic, 1 vCore, 2GiB Memory, 5GB storage",
```

Comment:
Can you match these labels to our SKU design pattern?

<img width="606" alt="Screen Shot 2021-06-11 at 4 12 54 PM" src="https://user-images.githubusercontent.com/22795803/121756864-729f8f80-cad0-11eb-9ad4-db360d16703e.png">

Here's the [source](https://github.com/microsoft/vscode-azuretools/blob/ba49322b5120c25f9202f0edd3383097afbab58d/appservice/src/createAppService/AppServicePlanSkuStep.ts#L66-L73) for that example. It also makes use of the `description` property for more info as needed.

To include more info about each tier, maybe adding a link similar to the "Show pricing information..." quick pick would work:

https://github.com/microsoft/vscode-azuretools/blob/ba49322b5120c25f9202f0edd3383097afbab58d/appservice/src/createAppService/AppServicePlanSkuStep.ts#L40

Reply:
And to clarify, I don't think you need to worry about the grouping stuff. Just tweaking the labels would be good

Reply:
This is what I settled on- -- 
<img width="748" alt="screenshot 2021-06-15 at 12 11 08" src="https://user-images.githubusercontent.com/1532417/121982503-f4463480-cdd2-11eb-83ce-d18f9befa110.png">